drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -2618,7 +2649,7 @@ static void rswitch_fib_event_add(struct rswitch_fib_event_work *fib_work)
 
 	dev = get_dev_by_ip(fib_work->priv, be32_to_cpu(nh->nh_saddr), false);
 	/* Do not offload routes, related to VMQs (etha equal to NULL) */
```

Comment:
Change comment due to changed logic (like "...equal to NULL and are not a VLAN devices")